zipkin-storage/cassandra/src/test/resources/simplelogger.properties

```diff
@@ -15,8 +15,8 @@ org.slf4j.simpleLogger.log.com.datastax.oss.driver.internal.core.cql.CqlRequestH
 org.slf4j.simpleLogger.log.com.datastax.oss.driver.internal.core.control.ControlConnection=error
 # ignore warnings about too many sessions
 org.slf4j.simpleLogger.log.com.datastax.oss.driver.internal.core.session.DefaultSession=error
-# stop huge spam
-org.slf4j.simpleLogger.log.org.testcontainers.dockerclient=off
-
-# Schema install takes a while. Log basic information to prevent CI from thinking things are hung
```

Comment:
this isn't really true anymore, so I matched logging config with the zipkin-dependencies stuff, despite this being simplelogger not log4j2